app/src/ui/history/selected-commits.tsx

```diff
@@ -497,7 +497,7 @@ export class SelectedCommits extends React.Component<
     showContextualMenu(items)
   }
 
-  private onViewOnGitHub = (sha: string, file: CommittedFileChange) => {
+  private onViewInBrowser = (sha: string, file: CommittedFileChange) => {
```

Comment:
There is no functionality change here. Can we return this to onViewOnGitHub since we are not switching to View on Browser in these instances?

Reply:
I think this is true in some other files as well, please update where applicable to remove unnecessary changes from this PR.